integration-test/src/main/java/uk/gov/gchq/gaffer/integration/impl/loader/schemas/SchemaLoader.java

```diff
@@ -39,6 +40,7 @@ public interface SchemaLoader {
     String B = "B";
     String C = "C";
     String D = "D";
+    @SuppressFBWarnings(value = "MS_OOI_PKGPROTECT", justification = "Value is not being changed")
     String[] VERTEX_PREFIXES = new String[]{A, B, C, D};
```

Comment:
Quite a few tests use this, could we reuse `AbstractStoreIT::VERTEX_PREFIXES`

Reply:
Value from `AbstractStoreIT` now used instead, this field has been removed and there's no longer a need to suppress this warning.